pymc/sampling_jax.py

```diff
@@ -94,6 +94,45 @@ def logp_fn_wrap(x):
     return logp_fn_wrap
 
 
+# Adopted from pm.to_inference_data
+def find_observations(model: Model) -> Dict[str, Any]:
```

Comment:
Can we refactor the function in the backends.arviz so that we don't duplicate the code inside the library.l? It seems that the original function can be taken out of the class where it exists or made into static method that expects a model as the only argument like here.

Then we can simply import it here.

Reply:
Yep, just refactored `find_observations` so we just import it